packages/vue-dot/playground/examples/DialogBoxEx.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+	<DocSection title="DialogBox">
```

Comment:
```suggestion
	<DocSection
		no-divider
		title="DialogBox"
	>
```
Comme tu l'as retiré de l'exemple précédent il ne faut pas oublier de le rajouter